<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: none;</string>
  </property>
  <widget class="QWidget" name="MainData" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>661</width>
     <height>651</height>
    </rect>
   </property>
   <widget class="QStackedWidget" name="Main">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>571</width>
      <height>571</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: rgba(237, 215, 214, 80);
color: white;
border-radius: 20px;
}

QPushButton::hover {
background: rgba(229, 232, 255, 40);
}

QStackedWidget{
border: 3px solid rgba(105, 105, 100, 70);
border-radius: 20px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="OrderNumber">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background: rgba(251, 239, 220, 15);
border-radius: 15px;
color: rgb(80, 77, 89);
selection-background-color: rgba(245, 245, 245, 0);
selection-color:  rgb(80, 77, 89);
}</string>
     </property>
     <widget class="QPushButton" name="backAccountBtn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>500</y>
        <width>144</width>
        <height>44</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>144</width>
        <height>44</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>190</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
font-size: 28px;
color: rgb(92, 89, 106);
}</string>
      </property>
      <property name="text">
       <string>Your order Number:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="showOrderNumber">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Profile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>55</height>
      </size>
     </property>
     <widget class="QPushButton" name="tripsBtn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>470</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>My Trips</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editBtn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>470</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Edit Account</string>
      </property>
     </widget>
     <widget class="QWidget" name="MainProfile" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>481</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background: rgba(251, 239, 220, 15);
border-radius: 15px;
color: rgb(80, 77, 89);
selection-background-color: rgba(245, 245, 245, 0);
selection-color:  rgb(80, 77, 89);
}

QLabel {
color: rgb(92, 89, 106);
}</string>
      </property>
      <widget class="QLabel" name="HelloLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>HELLO</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="EmailLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Your Email:</string>
       </property>
      </widget>
      <widget class="QLabel" name="PhoneNumberLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>300</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Your Phone Number:</string>
       </property>
      </widget>
      <widget class="QLabel" name="ProfileLogo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="res.qrc">:/images/images/profile.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="showUserName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>272</x>
         <y>37</y>
         <width>201</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="showEmail">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="ShowPhoneNumber">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>347</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="FindRide" native="true">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(229, 232, 255, 40);
color: white;
border-radius: 20px;
}

QPushButton::hover {
background: rgba(232, 216, 255, 128);

}
</string>
   </property>
   <widget class="QLabel" name="FindRideLogo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/images/rideCar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="findRideBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>FIND RIDE</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="CreateRide" native="true">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>360</y>
     <width>331</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(229, 232, 255, 40);
color: white;
border-radius: 20px;
}

QPushButton::hover {
background: rgba(232, 216, 255, 128);

}

</string>
   </property>
   <widget class="QLabel" name="CreateRideLogo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/images/images/createRide.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="createRideBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>CREAT RIDE</string>
    </property>
   </widget>
  </widget>
  <widget class="ClickableLabel" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>650</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
font: 18px;
color: rgb(241, 242, 241);
text-decoration: underline;
}

QLabel::hover {
color: rgba(242, 242, 220, 153);

}</string>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
